src/System.Runtime.Serialization.Formatters/src/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "frameworks": {
+    "netstandard1.4": {
+      "dependencies": {
+        "Microsoft.NETCore.Platforms": "1.0.1-rc3-24102-00",
+        "System.Collections": "4.0.10",
```

Comment:
SerializationInfo that I will fill in in a separate/next commit (as it has significant security related code) uses collections extensively. And so I left it.